cmd/mayactl/app/command/volume_info.go

```diff
@@ -294,7 +294,7 @@ Replica Count :   {{.ReplicaCount}}
 		// Confirming replica status, poolname , cvrName, nodeName are equal to replica count
 		if replicaCount != len(replicaStatus) || replicaCount != len(poolName) || replicaCount != len(cvrName) || replicaCount != len(nodeName) {
 			fmt.Println("Invalid response received from maya-api service")
-			return fmt.Errorf("%s", "Invalid response received from maya-api service")
+			return nil
 		}
```

Comment:
Have an else case as well to handle error cases.

Reply:
but which error to handle ? you mean to split the if cases to else if ?

Reply:
done

Reply:
Thanks !!